### PR TITLE
Generate flow type definition on build

### PR DESCRIPTION
**Pull request by existentialism**

### CODE REVIEW

**Comment by xtuc** [comment]
Has been released:
```
 - babel-plugin-transform-charcodes: 0.0.10 => 0.1.0
 - charcodes: 0.0.10 => 0.1.0
```

File: packages/babel-plugin-transform-charcodes/src/index.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// @flow
```
Comment (Andarist) [inline packages/babel-plugin-transform-charcodes/src/index.js]: @existentialism why did you remove the comment?

File: Makefile [hunk 2]
```diff
@@ -3,20 +3,24 @@ export PATH := $(PATH):./node_modules/.bin/
...
-	lerna bootstrap
+	./node_modules/.bin/lerna bootstrap
 	make build
 
 doc-charcodes:
 	dump-exports ./packages/charcodes/src/index.js > ./packages/charcodes/README.md
 
-build: doc-charcodes
+flow-charcodes:
+	./node_modules/.bin/flow gen-flow-files packages/charcodes/src --out-dir packages/charcodes/lib
```
Comment (Andarist) [inline Makefile]: nice, haven't heard of `gen-flow-files`, need to test this

File: packages/babel-plugin-transform-charcodes/src/index.js (hunk 1 above)
Comment (existentialism) [reply packages/babel-plugin-transform-charcodes/src/index.js]: @Andarist just temporary, mostly because it had a ton of flow errors, and it'd be primarily making everything `any`.
